### PR TITLE
TLSN: Added tags + improved description

### PR DESCRIPTION
**Pull request by heeckhau**

### CODE REVIEW

File: data/projects/tlsn.ts [hunk 1]
```diff
@@ -17,9 +21,19 @@ export const tlsn: ProjectInterface = {
     discord: "https://discord.gg/9XwESXtcN7",
   },
   tags: {
-    themes: ["build"],
-    type: [],
-    builtWith: [],
-    keywords: [],
+    themes: ["build", "play"],
+    type: [
```
Comment (kalidiagne) [inline data/projects/tlsn.ts:25]:
```suggestion
    types: [
```
please consider to add this change that will be required after this PR: https://github.com/privacy-scaling-explorations/website-v2/pull/87

**Comment by ChialiT** [comment]
@kichong let's review this one together. :)

File: data/projects/tlsn.ts [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+TLSNotary is ideal for developers of privacy-focused projects that require **data provenance** from secure web servers. It leverages the widely-used **Transport Layer Security (TLS)** protocol to securely and privately prove that a transcript of communications with a web server took place. The protocol divides TLS session keys between two parties: the Prover and the Verifier, using **Multi-Party Computation (MPC)**.  Neither the User nor Notary are in possession of the full TLS session keys, they only hold a share of those keys. This retains the security assumptions of TLS while allowing the Prover to demonstrate the **authenticity of the communication** to the Verifier. The Verifier remains unaware of which webserver is being queried, and the Verifier never has access to the unencrypted communications, except for the data the Prover explicitly wants to disclose.
+
+**TLSNotary can help your project with secure and privacy-preserving data portability!**
+
+[Learn more](https://tlsnotary.org/)
```
Comment (kichong) [inline data/projects/tlsn.ts:8]: remove this link since it's already on the page (at the top)

**Review by ChialiT — APPROVED** [review]

**Review by kichong — COMMENTED** [review]
looks great. just suggest removing the link so it matches the other descriptions.

**Review by kichong — APPROVED** [review]

**Pull request merged**